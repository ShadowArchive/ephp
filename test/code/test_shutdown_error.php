<?php

function teardown() {
    print "goodbye!\n";
    trigger_error("whoops!", E_USER_ERROR);
}

register_shutdown_function("teardown");

