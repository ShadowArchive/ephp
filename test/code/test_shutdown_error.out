goodbye!

Fatal error: whoops! in {{CWD}}/test/code/test_shutdown_error.php on line 5
